Homeworks/Homework_5/utils.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import sqlite3
+from functools import wraps
+import freecurrencyapi
+
+SELECT_COMMAND = 'SELECT {} FROM {} WHERE Id = {}'
+UPDATE_COMMAND = 'UPDATE Account SET Amount = Amount {} {} WHERE Id = {}'
```

Comment:
if you have some shared constants you need to create a separate file named constants.py))))
it's not utils.
but in my project I have a separate file called queries for such kind of templates.